frontend/packages/kubevirt-plugin/package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.0.0-fixed",
   "description": "Kubevirt - Virtual machines addon for Kubernetes",
   "private": true,
+  "scripts": {
+    "lint": "yarn --cwd ../.. lintme packages/kubevirt-plugin"
```

Comment:
This should be `yarn --cwd ../.. eslint packages/kubevirt-plugin`

This is similar to `yarn --cwd ../.. run test packages/console-plugin-sdk` in the plugin SDK package.

Reply:
updated to `eslint`